README.md

```diff
@@ -15,9 +15,9 @@ Report any bugs, improvements, or new feature requests at https://jira.mongodb.o
 Building Tools
 ---------------
 
-We currently build the tools with Go version 1.15, other Go versions may work but they are untested. `go get` will not
-work; you need to clone the repository to build it. Be sure to clone the repository into your Go workspace inside your 
-$GOPATH.
+We currently build the tools with Go version 1.15. Other Go versions may work but they are untested.
+
+Using `go get` will not work. To build the tools, you need to first clone this repository into your Go workspace inside your $GOPATH.
```

Comment:
I originally thought that switching to Go modules will allow others to download and build the tools with `go get`, but that's not the case.

If you do `go get github.com/mongodb/mongo-tools`, that'll download the tools just fine, but it'll also run `go install` which won't work since we have multiple binaries to build. So users still have to use the make script or run `build.go` themselves.

Reply:
That's too bad. Do you know what specific `go install` command it runs? When you say it "won't work" do you mean it does nothing or does it encounter an error?

Also, on a different note: since we've updated `mongo-tools` to modules, do we need to strictly clone the repo inside the `$GOPATH`?

Reply:
Specifically for building the tools — we haven't made the tools "go gettable" yet, but [judging from the docs](https://golang.org/cmd/go/#hdr-Add_dependencies_to_current_module_and_install_them), I believe `go get github.com/mongodb/mongo-tools` will download the repo's contents to $GOPATH/src and then run a plain `go install` from the repo's directory:
> With no package arguments and also without -u, 'go get' is not much more than 'go install'

I tried doing a plain `go install` / `go get` from the mongo-tools root directory and found that both basically make the build file into an executable named `mongo-tools`, and put it in $GOPATH/bin like so:

```
$ $GOPATH/bin/mongo-tools
USAGE: [tasks ...] [options ...]

TASKS:
  build(tools)
        build the tools
  ---------------------------------------------------------------------
  test -> [test:unit test:integration test:kerberos]
  test:integration(tools, ssl, auth, kerberos, topology)
        runs integration tests
...

$ $GOPATH/bin/mongo-tools build
START  | build
       | rm: bin/bsondump
       | exec: '/usr/local/bin/go run release/release.go get-version'
...
```

To summarize our options, I think these will all be equivalent ways to build the tools:
- `git clone github.com/mongodb/mongo-tools` + `./make build` from the cloned directory
- `go get github.com/mongodb/mongo-tools` + `$GOPATH/bin/mongo-tools build` from the $GOPATH/src/github.com/mongodb/mongo-tools directory
- `go get -d github.com/mongodb/mongo-tools` + `./make build` from the $GOPATH/src/github.com/mongodb/mongo-tools directory

So `go get` will technically work, but it seems kind of impractical to me to use it as a cloning mechanism for building the tools. It might be better suited to use in projects that need `mongo-tools` as a dependency.

If this all sounds right to you, do you think it's worth mentioning in the README?

Reply:
And to your second q — you're right that we won't need to clone inside the $GOPATH anymore. One of the many upsides of Go modules :)

I'll remove that phrase from the README.

Reply:
> If this all sounds right to you, do you think it's worth mentioning in the README?

Nah, not worth it! Thanks for the detailed explanation.


> And to your second q — you're right that we won't need to clone inside the $GOPATH anymore. One of the many upsides of Go modules :)
>
> I'll remove that phrase from the README.

Great